apps/vs-code-designer/src/app/commands/workflows/azureConnectorWizard.ts

```diff
@@ -107,3 +111,17 @@ class SaveAzureContext extends AzureWizardExecuteStep<IAzureConnectorsContext> {
     return context.enabled === false || !!context.subscriptionId || !!context.resourceGroup;
   }
 }
+
+//TODO: Update to be in webview after ignite redesign is done
+
+class ConnectorAuthenticationStep extends AuthenticationMethodSelectionStep<IAzureConnectorsContext> {
```

Comment:
Yep, same thing as with Andrew comment, this just seems to be a quite simple wrapper. Is there a way we can remove it and just call directly the AuthenticationMethodSelectionStep()?